src/components/Charts.jsx

```diff
@@ -2,52 +2,51 @@
 
 import React from 'react';
 
+import Immutable from 'immutable';
+
 import ChartWithHeader from './ChartWithHeader';
 
+const DIV_STYLE = {overflow: "auto", maxHeight: "100vh"};
+
 class Charts extends React.Component {
     render() {
-        var charts = [];
+        let charts = [];
```

Comment:
An assigned value from const will be only garbage collected when the lexical scopes dies. I am currently not sure on how this applies in situation like this.
Let on the other hand is block scoped and the value will be garbage collected after the block is closed.

Furthermore in this case I change the contents in the array. While the reference to the array might be considered constant the contents are not. 
Currently I would say that declaring charts as const is misleading?!